From e9a7cd1a16e2e21bc7e15eb009953c78beeeb3dc Mon Sep 17 00:00:00 2001
From: Allen Xu <b45815@freescale.com>
Date: Fri, 30 Jan 2015 01:57:44 +0800
Subject: [PATCH 0734/5242] MLK-10189: ARM:dts: disable gpmi nand module in
 imx6qdl sabreauto dts

commit  10c49d96a51360728104f571d7e4b7b071c7c975 from
https://source.codeaurora.org/external/imx/linux-imx.git

disable gpmi nand module in imx6qdl default dts since it conflicts with
uart3

Signed-off-by: Allen Xu <b45815@freescale.com>
(cherry picked from commit 54105e0ad2a4e30c5c048b5b9b233e49b4fba47d)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabreauto.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
index 2fc7ef2..cf3d50d 100644
--- a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
@@ -416,7 +416,7 @@
 &gpmi {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_gpmi_nand>;
-	status = "okay";
+	status = "disabled"; /* pin conflict with uart3 */
 	nand-on-flash-bbt;
 };
 
-- 
1.7.9.5

